am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/AccessManagement.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.amlib.models;
+
+import java.beans.ConstructorProperties;
+
+public class AccessManagement {
+
+    private final int accessManagementId;
+    private final String resourceId;
+    private final String accessorId;
+    private final int permissions;
+
+    @ConstructorProperties({"accessManagementId", "resourceId", "accessorId", "permissions"})
```

Comment:
isn't it done automatically when names are the same?

Reply:
We'd have to setup a flag for java compiler to emit field names in a particular way. That way we would also drop using @Bind annotation in *Repository methods. I don't know however the deeper implications of setting the flag so I decided to go with annotation.